### DISCUSSION TITLE
Proxy denies access from jackett?

### DISCUSSION THREAD

**Discussion by best-74** [haugene/docker-transmission-openvpn#2501] [Q&A]
Hi, i have an issue that my transmission proxy all of a sudden denies access from my jackett install. Any changes made recently to this?

I'm running without proxy login/pass since this is only internally available in the specific docker network.

**Comment by ilike2burnthing** [comment]
Last non-documentation related commits to master branch were from November, and they're minor. Dev branch's last commits are from the end of December.

**Reply by best-74** [reply]
Hmm, now i have an issue with this again.. seem to fall after a while.. wierd
